torch/onnx/utils.py

```diff
@@ -247,6 +248,14 @@ def _decide_constant_folding(do_constant_folding, operator_export_type):
     return _resolve_args_by_export_type("do_constant_folding", do_constant_folding, operator_export_type)
 
 
+def _decide_large_model_format(use_large_model_format, operator_export_type, f):
+    val_use_large_model_format = _resolve_args_by_export_type("use_large_model_format",
+                                                              use_large_model_format,
+                                                              operator_export_type)
+    model_file_location = f if val_use_large_model_format and isinstance(f, str) else str()
```

Comment:
f is the path when f is a string, otherwise, it's an empty string, in which case the location is the current repo? 
would it be possible to leave a short comment here?

Reply:
f must not be an empty string for large model export case. This is because you need an explicit path where to store parameter binary files. For regular sized model export case, f can be empty and it won't matter. For large model export, f must be a non-empty string specifying the path. This was chosen on purpose so as to not surprise the user with writing files in an unexpected location.

Reply:
Got it, thanks! 
Explaining this in use_large_model_format's doc and/or warning the user of this restriction on f with use_large_model_format=True on failure would be good

Reply:
Sure, I will add something in both places as you suggest.